### DISCUSSION TITLE
Fan Control will only actually run when 3DMark is running.

### DISCUSSION THREAD

**Discussion by 349078838** [Rem0o/FanControl.Releases#1772] [Q&A]
Two questions:
两个问题：
1.Fan Control will only actually run when 3DMark is running.
1.fan control 只有在3dmark运行时，才会真正运行。
2.2.The fan speed displayed on Fan Control is only displayed normally during the first debugging, and then it runs normally but does not display speed information, and can only view the speed information through other software.
2.Fan Control 上显示的风扇转速只有在第一次调试的时候显示是正常的，之后运转正常但是不显示转速信息，只能通过其他软件查看转速信息。
![image](https://github.com/Rem0o/FanControl.Releases/assets/56948466/fe9e92b4-2a88-481e-9703-1282d1a13fe6)
Other than that:
另外：
I do not understand English, and there are any irregularities in my speech, please let me know.
我不懂英文，发言有不规范之处，还请告知。
If you don't know if any other data is missing, please tell us how to extract it.
不知是否还缺少别的数据，请告知如何提取。
--------[ AIDA64 Extreme ]----------------------------------------------------------------------------------------------

    版本                                              AIDA64 v6.33.5700/cn
    检测模块                                          4.5.846.8-x64
    主页                                              http://www.aida64.com/
    报告类型                                          本地报告
    计算机                                            MITSUHIRO
    用户                                              Administrator
    操作系统                                          Microsoft Windows 10 Pro 10.0.19045.2913
    日期                                              2023-05-10
    时间                                              21:27

--------[ 系统概述 ]----------------------------------------------------------------------------------------------------

    计算机:
      计算机类型                                        基于 ACPI x64 的电脑
      操作系统                                          Microsoft Windows 10 Pro
      操作系统版本升级(SP)                              -
      Internet Explorer                                 11.789.19041.0
      Edge                                              112.0.1722.58
      DirectX                                           DirectX 12.0
      计算机名称                                        MITSUHIRO
      用户名称                                          Administrator
      登录域                                            MITSUHIRO
      日期/时间                                         2023-05-10 / 21:27

    主板:
      处理器名称                                        HexaCore AMD Ryzen 5 3600, 4150 MHz (41.5 x 100)
      主板名称                                          Asus TUF Gaming B450M-Pro S  (1 PCI-E x1, 2 PCI-E x16, 2 M.2, 4 DDR4 DIMM, Audio, Video, Gigabit LAN)
      主板芯片组                                        AMD B450, AMD K17.7 FCH, AMD K17.7 IMC
      系统内存                                          16294 MB
      DIMM3: Kingston HyperX KHX3200C16D4/8GX           8 GB DDR4-3000 DDR4 SDRAM  (18-17-17-36 @ 1501 MHz)  (17-17-17-36 @ 1501 MHz)  (16-17-17-36 @ 1501 MHz)  (15-17-17-36 @ 1501 MHz)  (14-16-16-34 @ 1401 MHz)  (13-15-15-32 @ 1301 MHz)  (12-14-14-29 @ 1201 MHz)  (11-13-13-27 @ 1101 MHz)  (10-12-12-24 @ 1001 MHz)  (9-11-11-22 @ 900 MHz)
      DIMM4: Kingston HyperX KHX3200C16D4/8GX           8 GB DDR4-3000 DDR4 SDRAM  (18-17-17-36 @ 1501 MHz)  (17-17-17-36 @ 1501 MHz)  (16-17-17-36 @ 1501 MHz)  (15-17-17-36 @ 1501 MHz)  (14-16-16-34 @ 1401 MHz)  (13-15-15-32 @ 1301 MHz)  (12-14-14-29 @ 1201 MHz)  (11-13-13-27 @ 1101 MHz)  (10-12-12-24 @ 1001 MHz)  (9-11-11-22 @ 900 MHz)
      BIOS 类型                                         AMI (08/10/2021)
      通讯端口                                          通信端口 (COM1)

    显示设备:
      显示适配器                                        Radeon RX 570 Series  (8 GB)
      显示适配器                                        Radeon RX 570 Series  (8 GB)
      显示适配器                                        Radeon RX 570 Series  (8 GB)
      显示适配器                                        Radeon RX 570 Series  (8 GB)
      显示适配器                                        Radeon RX 570 Series  (8 GB)
      显示适配器                                        Radeon RX 570 Series  (8 GB)
      显示适配器                                        Radeon RX 570 Series  (8 GB)
      3D 加速器                                         AMD Radeon RX 570 (Polaris 20)
      显示器                                            Dell P2415Q (DP)  [24" IPS LCD]  (3VV149AU09MB)
      显示器                                            Xiaomi Mi TV

    多媒体:
      音频适配器                                        ATI Radeon HDMI @ AMD Ellesmere - High Definition Audio Controller
      音频适配器                                        Realtek ALC1200 @ AMD K17.7 - High Definition Audio Controller

    存储设备:
      存储控制器                                        Microsoft 存储空间控制器
      存储控制器                                        StoreMI Bottom Device
      存储控制器                                        StoreMI Bottom Device
      存储控制器                                        StoreMI Controller [storport]
      存储控制器                                        StoreMI Controller [storport]
      硬盘驱动器                                        Hitachi HTS54101 SCSI Disk Device  (931 GB)
      硬盘驱动器                                        KIOXIA-EXCERIA P SCSI Disk Device  (465 GB)
      硬盘驱动器                                        ST3000DM007-1WY1 SCSI Disk Device  (3 TB, 5000 RPM, SATA-III)
      硬盘 SMART 状态                                   未知

    磁盘分区:
      C: (NTFS)                                         200.0 GB (57.0 GB 可用)
      D: (NTFS)                                         265.0 GB (144.4 GB 可用)
      E: (NTFS)                                         731.5 GB (447.3 GB 可用)
      F: (NTFS)                                         2056.0 GB (156.5 GB 可用)
      G: (NTFS)                                         738.5 GB (584.2 GB 可用)
      H: (NTFS)                                         200.0 GB (101.5 GB 可用)
      总大小                                            4191.0 GB (1490.9 GB 可用)

    输入设备:
      键盘                                              HID Keyboard Device
      键盘                                              HID Keyboard Device
      键盘                                              HID Keyboard Device
      键盘                                              HID Keyboard Device
      鼠标                                              HID-compliant mouse
      鼠标                                              HID-compliant mouse

    网络设备:
      主 IP 地址                                        192.168.50.163
      主 MAC 地址                                       24-4B-FE-CE-C8-E2
      网络适配器                                        Bluetooth Device (Personal Area Network)
      网络适配器                                        Intel(R) Wi-Fi 6 AX200 160MHz
      网络适配器                                        Microsoft Wi-Fi Direct Virtual Adapter #13
      网络适配器                                        Microsoft Wi-Fi Direct Virtual Adapter #14
      网络适配器                                        Realtek PCIe 2.5GbE Family Controller  (192.168.50.163)

    外围设备:
      打印机                                            Fax
      打印机                                            Microsoft Print to PDF
      打印机                                            Microsoft XPS Document Writer
      打印机                                            OneNote (Desktop)
      打印机                                            OneNote for Windows 10
      USB3 控制器                                       AMD K17.7 - USB 3.1 xHCI Controller
      USB3 控制器                                       AMD Low-Power Promontory F Chipset - USB xHCI Controller
      USB 设备                                          USB Composite Device
      USB 设备                                          USB Composite Device
      USB 设备                                          USB 输入设备
      USB 设备                                          USB 输入设备
      USB 设备                                          USB 输入设备
      USB 设备                                          USB 输入设备
      USB 设备                                          英特尔(R) 无线 Bluetooth(R)

    DMI:
      DMI BIOS 厂商                                     American Megatrends Inc.
      DMI BIOS 版本                                     3211
      DMI 系统制造商                                    ASUS
      DMI 系统产品                                      System Product Name
      DMI 系统版本                                      System Version
      DMI 系统序列号                                    System Serial Number
      DMI 系统 UUID                                     5383FDD1-8D03A204-2534244B-FECEC8E2
      DMI 主板制造商                                    ASUSTeK COMPUTER INC.
      DMI 主板产品                                      TUF GAMING B450M-PRO S
      DMI 主板版本                                      Rev X.0x
      DMI 主板序列号                                    200772436704370
      DMI 主机制造商                                    Default string
      DMI 主机版本                                      Default string
      DMI 主机序列号                                    Default string
      DMI 主机识别标签                                  Default string
      DMI 主机类型                                      Desktop Case

--------[ DMI ]---------------------------------------------------------------------------------------------------------

  [ BIOS ]

    BIOS 属性:
      厂商                                              American Megatrends Inc.
      版本                                              3211
      发布日期                                          08/10/2021
      大小                                              16 MB
      系统 BIOS 版本                                    5.17
      引导设备                                          Floppy Disk, Hard Disk, CD-ROM
      性能                                              Flash BIOS, Shadow BIOS, Selectable Boot, EDD, BBS
      支持的标准                                        DMI, APM, ACPI, UEFI
      扩展性能                                          PCI, USB
      虚拟机                                            否

    BIOS 制造商:
      公司名称                                          American Megatrends Inc.
      产品信息                                          https://ami.com/en/products/bios-uefi-firmware
      BIOS 升级                                         http://www.aida64.com/goto/?p=biosupdates

  [ 系统 ]

    系统:
      制造商                                            ASUS
      产品                                              System Product Name
      版本                                              System Version
      序列号                                            System Serial Number
      SKU#                                              SKU
      家族                                              To be filled by O.E.M.
      全局唯一识别码(UUID)                              5383FDD1-8D03A204-2534244B-FECEC8E2
      全局唯一识别码(UUID) (GUID)                       D1FD8353-038D-04A2-2534-244BFECEC8E2
      唤醒方式                                          电源开关

  [ 主板 ]

    主板:
      制造商                                            ASUSTeK COMPUTER INC.
      产品                                              TUF GAMING B450M-PRO S
      版本                                              Rev X.0x
      序列号                                            200772436704370
      标识                                              Default string

    主板制造商:
      公司名称                                          ASUSTeK Computer Inc.
      产品信息                                          https://www.asus.com/Motherboards
      BIOS 下载                                         https://www.asus.com/support
      驱动程序更新                                      http://www.aida64.com/goto/?p=drvupdates
      BIOS 升级                                         http://www.aida64.com/goto/?p=biosupdates

  [ 机箱 ]

    机型:
      制造商                                            Default string
      版本                                              Default string
      序列号                                            Default string
      标识                                              Default string
      机箱类型                                          台式机
      引导状态                                          安全
      电源状态                                          安全
      温度状态                                          安全
      安全性状态                                        无

  [ 处理器 / AMD Ryzen 5 3600 6-Core Processor ]

    处理器:
      制造商                                            Advanced Micro Devices, Inc.
      版本                                              AMD Ryzen 5 3600 6-Core Processor
      序列号                                            未知
      标识                                              未知
      部件号                                            未知
      外部时钟频率                                      100 MHz
      最大时钟频率                                      4200 MHz
      当前时钟频率                                      3600 MHz
      类型                                              Central Processor
      电压                                              1.1 V
      状态                                              已启用
      升级                                              Socket AM4
      Socket 定义                                       AM4
      HTT / CMP Units                                   2 / 6
      性能                                              64-bit, Multi-Core, Multiple Hardware Threads, Execute Protection, Enhanced Virtualization, Power/Performance Control

    CPU 制造商:
      公司名称                                          Advanced Micro Devices, Inc.
      产品信息                                          https://www.amd.com/en/products/processors-desktop
      驱动程序更新                                      http://www.aida64.com/goto/?p=drvupdates

  [ 高速缓存 / L1 - Cache ]

    高速缓存:
      类型                                              Unified
      速度                                              1 ns
      状态                                              已启用
      运行模式                                          Write-Back
      关联                                              8-way Set-Associative
      最大容量                                          384 KB
      已安装容量                                        384 KB
      支持 SRAM 类型                                    Pipeline Burst
      当前 SRAM 类型                                    Pipeline Burst
      错误修正                                          Multi-bit ECC
      Socket 定义                                       L1 - Cache

  [ 高速缓存 / L2 - Cache ]

    高速缓存:
      类型                                              Unified
      速度                                              1 ns
      状态                                              已启用
      运行模式                                          Write-Back
      关联                                              8-way Set-Associative
      最大容量                                          3 MB
      已安装容量                                        3 MB
      支持 SRAM 类型                                    Pipeline Burst
      当前 SRAM 类型                                    Pipeline Burst
      错误修正                                          Multi-bit ECC
      Socket 定义                                       L2 - Cache

  [ 高速缓存 / L3 - Cache ]

    高速缓存:
      类型                                              Unified
      速度                                              1 ns
      状态                                              已启用
      运行模式                                          Write-Back
      关联                                              16-way Set-Associative
      最大容量                                          32 MB
      已安装容量                                        32 MB
      支持 SRAM 类型                                    Pipeline Burst
      当前 SRAM 类型                                    Pipeline Burst
      错误修正                                          Multi-bit ECC
      Socket 定义                                       L3 - Cache

  [ 内存阵列 / System Memory ]

    内存阵列:
      位置                                              主板
      内存阵列功能                                      系统内存
      错误修正                                          无
      最大内存容量                                      128 GB
      内存设备                                          4

  [ 内存设备 / DIMM_A1 ]

    内存设备:
      设备定位                                          DIMM_A1
      Bank 定位                                         BANK 0
      制造商                                            未知
      序列号                                            未知
      部件号                                            未知
      Firmware Version                                  未知

  [ 内存设备 / DIMM_A2 ]

    内存设备:
      形状特征                                          DIMM
      类型                                              DDR4
      类型详情                                          Synchronous, Unbuffered
      大小                                              8 GB
      最大时钟频率                                      3200 MT/s
      当前时钟频率                                      3200 MT/s
      总带宽                                            64 位
      数据带宽                                          64 位
      Ranks                                             1
      最低电压                                          1.200 V
      最高电压                                          1.200 V
      当前电压                                          1.200 V
      设备定位                                          DIMM_A2
      Bank 定位                                         BANK 1
      制造商                                            Kingston
      序列号                                            4058352F
      部件号                                            KHX3200C16D4/8GX
      Memory Technology                                 DRAM
      Memory Operating Modes                            Volatile Memory
      Firmware Version                                  未知
      Module Manufacturer ID                            9801h
      Volatile Size                                     8 GB

  [ 内存设备 / DIMM_B1 ]

    内存设备:
      设备定位                                          DIMM_B1
      Bank 定位                                         BANK 2
      制造商                                            未知
      序列号                                            未知
      部件号                                            未知
      Firmware Version                                  未知

  [ 内存设备 / DIMM_B2 ]

    内存设备:
      形状特征                                          DIMM
      类型                                              DDR4
      类型详情                                          Synchronous, Unbuffered
      大小                                              8 GB
      最大时钟频率                                      3200 MT/s
      当前时钟频率                                      3200 MT/s
      总带宽                                            64 位
      数据带宽                                          64 位
      Ranks                                             1
      最低电压                                          1.200 V
      最高电压                                          1.200 V
      当前电压                                          1.200 V
      设备定位                                          DIMM_B2
      Bank 定位                                         BANK 3
      制造商                                            Kingston
      序列号                                            4B582791
      部件号                                            KHX3200C16D4/8GX
      Memory Technology                                 DRAM
      Memory Operating Modes                            Volatile Memory
      Firmware Version                                  未知
      Module Manufacturer ID                            9801h
      Volatile Size                                     8 GB

  [ 系统插槽 / PCIEX16_1 ]

    系统插槽:
      插槽名称                                          PCIEX16_1
      类型                                              PCI-E
      使用                                              正使用
      数据总线位宽                                      x16
      长度                                              长
      ID                                                0

  [ 系统插槽 / PCIEX16_2 ]

    系统插槽:
      插槽名称                                          PCIEX16_2
      类型                                              PCI-E
      使用                                              Available
      数据总线位宽                                      x4
      长度                                              长
      ID                                                1

  [ 系统插槽 / PCIEX1 ]

    系统插槽:
      插槽名称                                          PCIEX1
      类型                                              PCI-E
      使用                                              正使用
      数据总线位宽                                      x1
      长度                                              短
      ID                                                2

  [ 端口 / PS/2 Keyboard/Mouse ]

    端口:
      端口类型                                          Keyboard Port
      内部访问标识                                      PS/2 Keyboard/Mouse
      内部连接类型                                      无
      外部访问标识                                      PS/2 Keyboard/Mouse
      外部连接类型                                      PS/2

  [ 端口 / USB78 ]

    端口:
      端口类型                                          USB
      内部访问标识                                      USB78
      内部连接类型                                      无
      外部访问标识                                      USB78
      外部连接类型                                      USB

  [ 端口 / DP port ]

    端口:
      内部访问标识                                      DP
      内部连接类型                                      无
      外部访问标识                                      DP port

  [ 端口 / HDMI port ]

    端口:
      内部访问标识                                      HDMI
      内部连接类型                                      无
      外部访问标识                                      HDMI port

  [ 端口 / U31G1_3 ]

    端口:
      端口类型                                          USB
      内部访问标识                                      U31G1_3
      内部连接类型                                      无
      外部访问标识                                      U31G1_3
      外部连接类型                                      USB

  [ 端口 / U31G1_4 ]

    端口:
      端口类型                                          USB
      内部访问标识                                      U31G1_4
      内部连接类型                                      无
      外部访问标识                                      U31G1_4
      外部连接类型                                      USB

  [ 端口 / U31G1_5 ]

    端口:
      端口类型                                          USB
      内部访问标识                                      U31G1_5
      内部连接类型                                      无
      外部访问标识                                      U31G1_5
      外部连接类型                                      USB

  [ 端口 / U31G1_6 ]

    端口:
      端口类型                                          USB
      内部访问标识                                      U31G1_6
      内部连接类型                                      无
      外部访问标识                                      U31G1_6
      外部连接类型                                      USB

  [ 端口 / LAN ]

    端口:
      端口类型                                          Network Port
      内部访问标识                                      LAN
      内部连接类型                                      无
      外部访问标识                                      LAN
      外部连接类型                                      RJ-45

  [ 端口 / U32G2_1 ]

    端口:
      端口类型                                          USB
      内部访问标识                                      U32G2_1
      内部连接类型                                      无
      外部访问标识                                      U32G2_1
      外部连接类型                                      USB

  [ 端口 / U32G2_C2 ]

    端口:
      端口类型                                          USB
      内部访问标识                                      U32G2_C2
      内部连接类型                                      无
      外部访问标识                                      U32G2_C2
      外部连接类型                                      USB

  [ 端口 / AUDIO ]

    端口:
      端口类型                                          Audio Port
      内部访问标识                                      AUDIO
      内部连接类型                                      无
      外部访问标识                                      AUDIO

  [ 端口 / BIOS_Flashback ]

    端口:
      内部访问标识                                      BIOS_Flashback
      内部连接类型                                      无
      外部访问标识                                      BIOS_Flashback

  [ 端口 / SATA6G_1 ]

    端口:
      端口类型                                          SATA
      内部访问标识                                      SATA6G_1
      内部连接类型                                      SATA/SAS Plug Receptacle
      外部连接类型                                      无

  [ 端口 / SATA6G_2 ]

    端口:
      端口类型                                          SATA
      内部访问标识                                      SATA6G_2
      内部连接类型                                      SATA/SAS Plug Receptacle
      外部连接类型                                      无

  [ 端口 / SATA6G_3 ]

    端口:
      端口类型                                          SATA
      内部访问标识                                      SATA6G_3
      内部连接类型                                      SATA/SAS Plug Receptacle
      外部连接类型                                      无

  [ 端口 / SATA6G_4 ]

    端口:
      端口类型                                          SATA
      内部访问标识                                      SATA6G_4
      内部连接类型                                      SATA/SAS Plug Receptacle
      外部连接类型                                      无

  [ 端口 / M.2_1 ]

    端口:
      端口类型                                          SATA
      内部访问标识                                      M.2_1
      内部连接类型                                      SATA/SAS Plug Receptacle
      外部连接类型                                      无

  [ 端口 / M.2_2 ]

    端口:
      端口类型                                          SATA
      内部访问标识                                      M.2_2
      内部连接类型                                      SATA/SAS Plug Receptacle
      外部连接类型                                      无

  [ 端口 / USB910 ]

    端口:
      端口类型                                          USB
      内部访问标识                                      USB910
      内部连接类型                                      USB
      外部连接类型                                      无

  [ 端口 / USB1112 ]

    端口:
      端口类型                                          USB
      内部访问标识                                      USB1112
      内部连接类型                                      USB
      外部连接类型                                      无

  [ 端口 / CPU_FAN ]

    端口:
      内部访问标识                                      CPU_FAN
      外部连接类型                                      无

  [ 端口 / CPU_OPT ]

    端口:
      内部访问标识                                      CPU_OPT
      外部连接类型                                      无

  [ 端口 / CHA_FAN1 ]

    端口:
      内部访问标识                                      CHA_FAN1
      外部连接类型                                      无

  [ 端口 / COM ]

    端口:
      端口类型                                          Serial Port 16550A Compatible
      内部访问标识                                      COM
      内部连接类型                                      DB-9 pin male
      外部连接类型                                      无

  [ 端口 / AAFP ]

    端口:
      端口类型                                          Audio Port
      内部访问标识                                      AAFP
      内部连接类型                                      Mini-jack (headphones)
      外部连接类型                                      无

  [ 端口 / F_PANEL ]

    端口:
      内部访问标识                                      F_PANEL
      外部连接类型                                      无

  [ 端口 / SPEAKER ]

    端口:
      内部访问标识                                      SPEAKER
      外部连接类型                                      无

  [ 端口 / CHA_FAN2 ]

    端口:
      内部访问标识                                      CHA_FAN2
      外部连接类型                                      无

  [ 端口 / TPM ]

    端口:
      内部访问标识                                      TPM
      外部连接类型                                      无

  [ 端口 / RGB_HEADER1 ]

    端口:
      内部访问标识                                      RGB_HEADER1
      外部连接类型                                      无

  [ 端口 / RGB_HEADER2 ]

    端口:
      内部访问标识                                      RGB_HEADER2
      外部连接类型                                      无

  [ 主板集成设备 / To Be Filled By O.E.M. ]

    主板集成设备:
      描述                                              To Be Filled By O.E.M.
      类型                                              Video
      状态                                              已启用

  [ 主板集成设备 / Onboard IGD ]

    主板集成设备:
      描述                                              Onboard IGD
      类型                                              Video
      状态                                              已启用
      总线/设备/功能                                    0 / 2 / 0

  [ 管理设备 / ITE IT8655E ]

    管理设备:
      描述                                              ITE IT8655E
      Base Address (Port)                               0295h

  [ TPM Device ]

    TPM 设备:
      制造商 ID                                         AMD
      TPM 版本                                          2.0
      描述                                              AMD
      Family Configurable via Firmware Update           不支持
      Family Configurable via Platform Software Support 支持
      Family Configurable via OEM Proprietary Mechanism 不支持

  [ 其它 ]

    其它:
      OEM String                                        Default string
      OEM String                                        Default string
      OEM String                                        SASHIMI
      OEM String                                        Default string
      OEM String                                        Default string
      OEM String                                        Default string
      OEM String                                        Default string
      OEM String                                        Default string
      System Configuration Option                       Default string
      System Configuration Option                       Default string
      System Configuration Option                       Default string
      System Configuration Option                       Default string

--------[ 超频 ]--------------------------------------------------------------------------------------------------------

    中央处理器(CPU):
      处理器名称                                        HexaCore AMD Ryzen 5 3600
      内部名称                                          Matisse
      制程步进                                          MTS-B0
      Engineering Sample                                否
      名称                                              AMD Ryzen 5 3600 6-Core Processor
      修订版本                                          00870F10h
      CPU VID                                           1.1000 V

    中央处理器速度:
      CPU 核心频率                                      3320.2 MHz
      CPU 倍频                                          33.2x
      CPU 外频(FSB)                                     100.0 MHz  (原始频率： 100 MHz)
      北桥时钟频率                                      1600.1 MHz
      内存总线频率                                      1600.1 MHz
      DRAM : FSB                                        48:3

    CPU 高速缓存:
      L1 代码缓存                                       32 KB per core
      L1 数据缓存                                       32 KB per core
      L2 缓存                                           512 KB per core  (On-Die, ECC, Full-Speed)
      L3 缓存                                           32 MB

    主板:
      主板 ID                                           63-0100-000001-00101111-122418-Chipset$0AAAA000_BIOS DATE: 08/10/21 05:16:53 VER: 05.00011
      主板名称                                          Asus TUF Gaming B450M-Pro S  (1 PCI-E x1, 2 PCI-E x16, 2 M.2, 4 DDR4 DIMM, Audio, Video, Gigabit LAN)

    芯片组:
      主板芯片组                                        AMD B450, AMD K17.7 FCH, AMD K17.7 IMC
      内存计时                                          16-18-18-36  (CL-RCD-RP-RAS)
      Command Rate (CR)                                 1T
      DIMM3: Kingston HyperX KHX3200C16D4/8GX           8 GB DDR4-3000 DDR4 SDRAM  (18-17-17-36 @ 1501 MHz)  (17-17-17-36 @ 1501 MHz)  (16-17-17-36 @ 1501 MHz)  (15-17-17-36 @ 1501 MHz)  (14-16-16-34 @ 1401 MHz)  (13-15-15-32 @ 1301 MHz)  (12-14-14-29 @ 1201 MHz)  (11-13-13-27 @ 1101 MHz)  (10-12-12-24 @ 1001 MHz)  (9-11-11-22 @ 900 MHz)
      DIMM4: Kingston HyperX KHX3200C16D4/8GX           8 GB DDR4-3000 DDR4 SDRAM  (18-17-17-36 @ 1501 MHz)  (17-17-17-36 @ 1501 MHz)  (16-17-17-36 @ 1501 MHz)  (15-17-17-36 @ 1501 MHz)  (14-16-16-34 @ 1401 MHz)  (13-15-15-32 @ 1301 MHz)  (12-14-14-29 @ 1201 MHz)  (11-13-13-27 @ 1101 MHz)  (10-12-12-24 @ 1001 MHz)  (9-11-11-22 @ 900 MHz)

    BIOS 属性:
      系统 BIOS 日期                                    08/10/2021
      视频 BIOS 日期                                    03/18/19
      DMI BIOS 版本                                     3211

    图形处理器(GPU):
      显示适配器                                        AMD Radeon RX 570 (Polaris 20)
      GPU 代码名称                                      Polaris 20 XL  (PCI Express 3.0 x16 1002 / 67DF, Rev EF)
      GPU 核心频率                                      1256 MHz  (原始频率： 1256 MHz)
      显存频率                                          1750 MHz  (原始频率： 1750 MHz)

--------[ 电源管理 ]----------------------------------------------------------------------------------------------------

    电源管理:
      当前电源                                          交流电源
      电池状态                                          无电池
      电池寿命                                          未知
      剩余寿命                                          未知

--------[ 便携式计算机 ]------------------------------------------------------------------------------------------------

    Centrino (Carmel) 兼容平台:
      中央处理器(CPU): Intel Pentium M (Banias/Dothan)  否  (AMD Ryzen 5 3600)
      芯片组: Intel i855GM/PM                           否  (AMD B450, AMD K17.7 FCH, AMD K17.7 IMC)
      WLAN: Intel PRO/Wireless                          否
      系统: 兼容 Centrino                               否

    Centrino (Sonoma) 兼容平台:
      中央处理器(CPU): Intel Pentium M (Dothan)         否  (AMD Ryzen 5 3600)
      芯片组: Intel i915GM/PM                           否  (AMD B450, AMD K17.7 FCH, AMD K17.7 IMC)
      WLAN: Intel PRO/Wireless 2200/2915                否
      系统: 兼容 Centrino                               否

    Centrino (Napa) 兼容平台:
      中央处理器(CPU): Intel Core (Yonah) / Core 2 (Merom)否  (AMD Ryzen 5 3600)
      芯片组: Intel i945GM/PM                           否  (AMD B450, AMD K17.7 FCH, AMD K17.7 IMC)
      WLAN: Intel PRO/Wireless 3945/3965                否
      系统: 兼容 Centrino                               否

    Centrino (Santa Rosa) 兼容平台:
      中央处理器(CPU): Intel Core 2 (Merom/Penryn)      否  (AMD Ryzen 5 3600)
      芯片组: Intel GM965/PM965                         否  (AMD B450, AMD K17.7 FCH, AMD K17.7 IMC)
      WLAN: Intel Wireless WiFi Link 4965               否
      系统: 兼容 Centrino                               否

    Centrino 2 (Montevina) 兼容平台:
      中央处理器(CPU): Intel Core 2 (Penryn)            否  (AMD Ryzen 5 3600)
      芯片组: Mobile Intel 4 Series                     否  (AMD B450, AMD K17.7 FCH, AMD K17.7 IMC)
      WLAN: Intel WiFi Link 5000 Series                 否
      系统: 兼容 Centrino 2                             否

    Centrino (Calpella) 兼容平台:
      中央处理器(CPU): Intel Core i3/i5/i7 (Arrandale/Clarksfield)否  (AMD Ryzen 5 3600)
      芯片组: Mobile Intel 5 Series                     否  (AMD B450, AMD K17.7 FCH, AMD K17.7 IMC)
      WLAN: Intel Centrino Advanced-N / Ultimate-N / Wireless-N否
      系统: 兼容 Centrino                               否

    Centrino (Huron River) 兼容平台:
      中央处理器(CPU): Intel Core i3/i5/i7 (Sandy Bridge-MB)否  (AMD Ryzen 5 3600)
      芯片组: Mobile Intel 6 Series                     否  (AMD B450, AMD K17.7 FCH, AMD K17.7 IMC)
      WLAN: Intel Centrino Advanced-N / Ultimate-N / Wireless-N否
      系统: 兼容 Centrino                               否

    Centrino (Chief River) 兼容平台:
      中央处理器(CPU): Intel Core i3/i5/i7 (Ivy Bridge-MB)否  (AMD Ryzen 5 3600)
      芯片组: Mobile Intel 7 Series                     否  (AMD B450, AMD K17.7 FCH, AMD K17.7 IMC)
      WLAN: Intel Centrino Advanced-N / Ultimate-N / Wireless-N否
      系统: 兼容 Centrino                               否

    Centrino (Shark Bay-MB) 兼容平台:
      中央处理器(CPU): Intel Core i3/i5/i7 (Haswell-MB) 否  (AMD Ryzen 5 3600)
      芯片组: Mobile Intel 8/9 Series                   否  (AMD B450, AMD K17.7 FCH, AMD K17.7 IMC)
      WLAN: Intel Centrino Advanced-N / Ultimate-N / Wireless-N否
      系统: 兼容 Centrino                               否

--------[ 传感器 ]------------------------------------------------------------------------------------------------------

    传感器:
      传感器类型                                        ITE IT8655E  (MMIO F7000000h)
      GPU 传感器类型                                    Diode, Asus ASP1106  (ATI-Diode, 20h)
      主板名称                                          Asus A320M / B350-Pro / B350M-Dragon / B450M-D3V / B450M-Dragon / EX-A320M / G11DF / Prime A320 / Prime B350 / Prime B450 / Prime X370 / Pro A320M / ROG Strix X470-I / TUF B350 / TUF B450 / TUF X470 S

    温度:
      主板                                              41 °C
      中央处理器(CPU)                                   59 °C
      CPU 二极管                                        61 °C
      GPU 二极管                                        78 °C

    冷却风扇:
      中央处理器(CPU)                                   359 RPM
      #1 机箱                                           1923 RPM
      #2 机箱                                           1683 RPM
      图形处理器(GPU)                                   1811 RPM  (54%)

    电压:
      CPU 核心                                          1.406 V
      CPU VID                                           1.100 V
      +3.3 V                                            3.357 V
      +5 V                                              5.041 V
      +12 V                                             12.099 V
      CPU VDD                                           1.363 V
      CPU VDDNB                                         1.094 V
      GPU 核心                                          1.031 V

    电流:
      CPU VDD                                           28.06 A
      CPU VDDNB                                         9.38 A
      GPU VRM                                           82.92 A

    功耗:
      CPU Package                                       38.10 W
      CPU VDD                                           38.23 W
      CPU VDDNB                                         10.26 W
      图形处理器(GPU)                                   116.78 W
      GPU VRM                                           85.51 W

**Comment by Rem0o** [comment]
![image](https://github.com/Rem0o/FanControl.Releases/assets/14118956/57a82544-a1fe-4ef4-b0d2-2c0465796b90)
